src/cli/main.cpp

```diff
@@ -37,8 +37,8 @@ std::string main_help()
 int main(int argc, char* argv[])
    {
    std::cerr << Botan::runtime_version_check(BOTAN_VERSION_MAJOR,
-                                             BOTAN_VERSION_MINOR,
-                                             BOTAN_VERSION_PATCH);
+             BOTAN_VERSION_MINOR,
+             BOTAN_VERSION_PATCH);
 
```

Comment:
You are right ;) There could be this kind of deviations - astyle is not so inteligent - however it's easy to fix to follow 1) or 2) just one more rule is needed. I will fix it. and prepare self review here for this kind of issues ;)